src/pages/sign-up/SignIn.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+function SignUp() {
+  return <h1>SignUp</h1>
+}
+
+export default SignUp
```

Comment:
change the component file name also to SignUp

Reply:
done